matlab/MeCorr/methylVariables.m

```diff
@@ -0,0 +1,70 @@
+% Plot correlation values between AUC of growth inhibition of cell 
+% lines and methylation flux
+
+% Workflow: 1)Run make_eGEM 2)Run this script 3)Assign epsilon_methylation
+% 3)Run last module of MATLAB_CODE_methyl (3 sections)
+
+% initCobraToolbox;
+% changeCobraSolver('gurobi');
+
+% s3 contains auc data. 
+s3= readtable('Data\Ludlow2015_SmallMolecInformer.xlsx','Sheet','s3');
+s3= table2array(s3);
+% s2 matches cell line name to number assigned in this study
+s2= readtable('Data\Ludlow2015_SmallMolecInformer.xlsx','Sheet','s2');
+
+%The KMTi's and KDMi's in Ludlow data s3
+%index_KMTi= [3, 178, 280, 374, 380, 421, 431, 475];
+%hmei_list= {'BRD-A02303741','BIX-01294','methylstat','QW-BI-011',...
+%    'UNC0321','CBB-1007','UNC0638','GSK-J4'};
+% drugs 280, 421, and 475 are KDMi's. Others are KMTi's
+%index_DNMTi = [2 , 78, 181, 402, 444];     % DNMT = DNA MT
+
+% Get all 3 rows of s3 for drug_auc_me. Replaces drug_auc_expt
+drug_auc_me= zeros(664*8, 3);

```

Comment:
Good first start -> Need to figure out way to extract via string or some other identifier.